CSharpCodeGenerator/CSharpCodeGenerator/CSharpCodeGenerator.csproj

```diff
@@ -1,22 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
     <ApplicationIcon />
     <OutputType>Exe</OutputType>
-    <StartupObject />
-    <LangVersion>8.0</LangVersion>
     <SelfContained>false</SelfContained>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="CommandLineParser" Version="2.6.0" />
-    <PackageReference Include="Improbable.CSharpCodeGen" Version="0.0.1-preview" />
-    <PackageReference Include="Improbable.Stdlib.CSharpCodeGen" Version="0.0.2-preview" />
-    <PackageReference Include="Improbable.Postgres.CSharpCodeGen" Version="0.0.2-preview" />
-    <PackageReference Include="Improbable.DatabaseSync.CSharpCodeGen" Version="0.0.1-preview" />
-    <PackageReference Include="Improbable.WorkerSdkInterop.CSharpCodeGen" Version="0.0.1-preview" />
-    <PackageReference Include="Improbable.Schema.Bundle" Version="0.0.2-preview" />
+    <PackageReference Include="CommandLineParser" Version="2.4.3" />
```

Comment:
Why was this downgraded?

Reply:
Copy-pasta, I'll fix it up across the board